src/pl/senla/hotel/configuration/AppPropertiesLoader.java

```diff
@@ -0,0 +1,71 @@
+package pl.senla.hotel.configuration;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+
+import static pl.senla.hotel.configuration.AppConfiguration.*;
+import static pl.senla.hotel.constant.InputOutputConstant.*;
+
+public class AppPropertiesLoader implements PropertiesLoader {
+
+    Properties appProperties = new Properties();
+    File propertiesFile = new File("C://IT/Properties/hotel.properties");
+
+    @Override
+    public Map<String, String> loadConfiguration() {
+        Map<String, String> configuration = new HashMap<>();
+        try (FileInputStream fis = new FileInputStream(propertiesFile)) {
+            appProperties.load(fis);
+            configuration.put(KEY_ABLE_TO_CHANGE_ROOM_STATUS,
```

Comment:
I did exactly the same:
1. I prepare prorties file in separate method, but this method is called in Main. This method is called only the first time, so now I commented it in Main.
2. I read properties during starting of application and before reading all data from saved files.

Is it right way of using properties-file?

Reply:
all loaded properties I put to Map and later use values from Map (receive by keys) in neccesery places.

Could I do this by this  way  or will be better implementation without Map and read values directly from properties-file?

Reply:
This and the previous comment about the constructor should be considered together.

You can read your file and prepare all the properties for use immediately when starting the program, because it is understood that the file cannot be changed while the program is running.

Therefore, it is better to transfer the file proofreading logic to the constructor of the class that will operate with them, i.e. has Properties.java.

Additionally, it seems to me that it is unnecessary to create an additional map for properties; the Properties field itself is enough. If you look inside this class, you can see that they are already stored in the map, but with a lot of ready-made functionality with checks and receiving.

It is enough to simply load the values into Properies during the execution of the constructor, and make methods for obtaining values by field names, for example:

```
public Integer getIntProperty(String propertyname) {
return Integer.parseInt(appProperties.getProperty(propertyname));
}

public Boolean getIntProperty(String propertyname) {
return Boolean.parseBoolean(appProperties.getProperty(propertyname));
}
```

Reply:
Also I can't find you properties file? Did you pushed it into git-repo?

Reply:
> Also I can't find you properties file. Did you pushed it into git-repo?

my mistake: saved it in separate directory. Now added in my project  a new package Resources and put hotel.properties there.

Reply:
> 
> Additionally, it seems to me that it is unnecessary to create an additional map for properties, the Properties field itself is enough. If you look inside this class, you can see that they are already stored in the map, but with a lot of ready-made functionality with checks and receiving.
> 

I agree with it, my mistake.
I rewrote without Map.

Reply:
> It is enough to simply load the values into Properies during the execution of the constructor, and make methods for obtaining values by properties names, for example:

It has been implemented here:  pl.senla.hotel.configuration.AppConfiguration.
I've just edited it for using Properties (instead of unnecessary using of Map).

Reply:
ready.
I hope I've done it correctly.